src/ripple/app/hook/impl/applyHook.cpp

```diff
@@ -3640,28 +5573,33 @@ DEFINE_HOOK_FUNCTION(
         memory,
         memory_length);
 
-    HOOK_TEARDOWN();
+    WASM_HOOK_TEARDOWN();
 }
 
-// Deterministic nonces (can be called multiple times)
-// Writes nonce into the write_ptr
-DEFINE_HOOK_FUNCTION(
-    int64_t,
-    etxn_nonce,
-    uint32_t write_ptr,
-    uint32_t write_len)
+DEFINE_JS_FUNCNARG(
+    JSValue,
+    hook_account);
```

Comment:
Unnecessary semicolon
```suggestion
    hook_account)
```

Reply:
https://github.com/Xahau/xahaud/pull/481